var/spack/repos/builtin/packages/py-bitstruct/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBitstruct(PythonPackage):
+    """This module is intended to have a similar interface as the python
+    struct module, but working on bits instead of primitive data types
+    (char, int, …)"""
```

Comment:
```suggestion
    (char, int, ...)"""
```
Not important since we no longer support Python 2 but unicode chars always scare me in docstrings.